.github/workflows/ci-build.yaml

```diff
@@ -140,18 +140,19 @@ jobs:
       - run: docker load < /tmp/argoexec_image.tar
       - name: Set-up /etc/hosts
         run: |
-          echo '127.0.0.1 dex'      | sudo tee -a /etc/hosts
-          echo '127.0.0.1 minio'    | sudo tee -a /etc/hosts
-          echo '127.0.0.1 postgres' | sudo tee -a /etc/hosts
-          echo '127.0.0.1 mysql'    | sudo tee -a /etc/hosts
+          echo '127.0.0.1 dex'              | sudo tee -a /etc/hosts
+          echo '127.0.0.1 minio'            | sudo tee -a /etc/hosts
+          echo '127.0.0.1 postgres'         | sudo tee -a /etc/hosts
+          echo '127.0.0.1 mysql'            | sudo tee -a /etc/hosts
+          echo '127.0.0.1 devstoreaccount1' | sudo tee -a /etc/hosts
```

Comment:
devstoreaccount1 is the default account for azurite, which simulates blob storage. The CI tests set AZURE=true to run e2e tests that exercise the new code hitting the azurite instance instead of actual blob storage.

Reply:
I was able to change the name without too much trouble, so I updated it to azurite.